app/routes/application.js

```diff
@@ -38,6 +38,9 @@ export default Ember.Route.extend({
       if(reason.name === "QuotaExceededError") {
         this.get("logger").error(reason);
         this.get("messageBox").alert(this.get("i18n").t("QuotaExceededError"));
+      } else if (reason.name === "NotFoundError" && reason.code === 8) {
```

Comment:
Catches "Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node."